code/game/objects/items/stacks/stack.dm

```diff
@@ -178,7 +178,7 @@
 			recipes_list = srl.recipes
 		var/datum/stack_recipe/R = recipes_list[text2num(href_list["make"])]
 		var/multiplier = text2num(href_list["multiplier"])
-		if (!multiplier ||(multiplier <= 0)) //href protection
+		if (!multiplier || !isnum_safe(multiplier) || (multiplier <= 0)) //href protection
```

Comment:
I believe the !multiplier is unnecessary now, since the !isnum_safe would handle if the value is null when it checks if it is a number

Reply:
And the multiplier <= 0 handles if the value is actually 0